server/client_events/client_chat_event.py

```diff
@@ -0,0 +1,5 @@
+import json
+from server.server_events import server_chat_event
+
+def handle(data, source_user):
+    server_chat_event.fire("{1}: {0}".format(data, source_user))
```

Comment:
i don't think it's a great idea to just send the message over without doing any kind of validation at all, if i wanted to annoy users by sending a message 30000 chars long, there'd be nothing to stop me from doing that here.

(i'll add this on my own after merge, you don't need to do anything here)